base/linalg/eigen.jl

```diff
@@ -164,6 +164,28 @@ Same as [`eigvals`](@ref), but saves space by overwriting the input `A`, instead
 The option `permute=true` permutes the matrix to become
 closer to upper triangular, and `scale=true` scales the matrix by its diagonal elements to
 make rows and columns more equal in norm.
+
+!!! note
+    The input array `A` will not contain its eigenvalues after `eigvals!` is called
+    on it - it is used as a workspace instead.
```

Comment:
Perhaps "matrix" instead of "array" to be consistent with the text above? Also, does the "instead" sound a bit odd? Instead of what? Could write it as 

> ... on it - `A` is only used as a workspace

or something along those lines?